modules/geo-layers/src/tile-layer/tile-layer.js

```diff
@@ -96,6 +99,12 @@ export default class TileLayer extends CompositeLayer {
       this.state.tileset.tiles.forEach(tile => {
         tile.layers = null;
       });
+    } else if (changeFlags.viewportChanged) {
+      if (tileset.isLoaded && onViewportChange) {
+        onViewportChange(tileset.selectedTiles.map(tile => tile.data), {
```

Comment:
- Add to documentation
- Drop the first argument for perf & usability. This is a new api and it does not have to be aligned with anything else.

Reply:
Done